iOS Client/Models/MBDevotion.swift

```diff
@@ -58,7 +58,7 @@ struct LoadedDevotion: Codable, Detailable {
             return nil
         }
         var dateComponents = DateComponents()
-        dateComponents.hour = 10
+        dateComponents.hour = 12
```

Comment:
haha let's go! cool man

Reply:
I guess all the stuff about time zones will "just work" Idk. we'll see. Maybe some localized logic is somewhere